torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2601,6 +2626,14 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                        SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_hard', device_type='cpu',
                                 dtypes=[torch.cfloat, torch.cdouble], active_if=IS_MACOS),
                    )),
+    OpInfo('cov',
+           dtypes=all_types_and_complex(),
+           sample_inputs_func=sample_inputs_cov,
+           skips=(
+           # "cov" not implemented for 'BFloat16'
+              SkipInfo('TestOpInfo', 'test_supported_dtypes',
```

Comment:
Instead of skipping `test_supported_dtypes` you should be able to indicate the dtypes that cov supports does not include bfloat16 using `dtypes` or `dtypesIfCPU` and `dtypesIfCUDA`